libs/SalesforceSDKCore/SalesforceSDKCore.xcodeproj/project.pbxproj

```diff
@@ -91,6 +91,7 @@
 		82D4642619C7C8170006BDFE /* CoreGraphics.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8280EBCC16E14F7000768DE8 /* CoreGraphics.framework */; };
 		82D4642719C7C8170006BDFE /* UIKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 8280EBB516E14E9F00768DE8 /* UIKit.framework */; };
 		B7394AE91D334D7400248DBE /* WebKit.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = B7394AE81D334D7400248DBE /* WebKit.framework */; };
+		BC9BFF421D5A46B6009FFA1F /* test_credentials.json in Resources */ = {isa = PBXBuildFile; fileRef = BC9BFF411D5A46B6009FFA1F /* test_credentials.json */; };
```

Comment:
SalesforceSDKIdentityTests tests were failing because we forgot to put test_credentials.json into the bundle..